README_AWS.md

```diff
@@ -51,7 +51,11 @@ OSX:
 Test The Setup
 --------------
 1. cd openshift-ansible
-1. Try to list all instances:
+1. To create a cluster with one master and two nodes:
```

Comment:
Agreed, I'll see about pulling https://github.com/openshift/openshift-ansible/pull/125 in as well so there is a list action we can use for testing.